projects/transformers/experiments/bertitos.py

```diff
@@ -47,14 +47,23 @@
 # |--------------------------------------------------------------------------|
 # | model           | num params | train loss  | eval loss      | perplexity |
 # |-----------------|:----------:|:-----------:|:--------------:|:----------:|
-# | small_bert_100k | 27,523,072 | 2.951       | 3.861          | 14.532     |
-# | small_bert_50k  | 27,523,072 | 3.373       | 4.417          | 21.386     |
-# | mini_bert_100k  | 10,615,808 | 3.669       | 4.773          | 27.342     |
-# | mini_bert_50k   | 10,615,808 | 4.464       | 5.815          | 56.309     |
-# | tiny_bert_100k  |  4,124,928 | 4.566       | 5.802          | 55.79276   |
-# | tiny_bert_50k   |  4,124,928 | 5.990       | 8.367          | 330.234    |
+# | small_bert_100k | 27,523,072 | 2.951       | 2.676          | 14.532     |
+# | small_bert_50k  | 27,523,072 | 3.373       | 3.063          | 21.386     |
+# | mini_bert_100k  | 10,615,808 | 3.669       | 3.308          | 27.342     |
+# | mini_bert_50k   | 10,615,808 | 4.464       | 4.031          | 56.309     |
```

Comment:
I assumed you were using log base 2, which seemed odd. This change means we should use natural logarithm instead for the transformation, right?

Reply:
Doesn't it seem odd to you that this leaves eval_loss lower than train_loss in all cases? This is highly unusual in machine learning, specially with this consistency and with undertrained models.

Reply:
Correct